root/static/scripts/common.js

```diff
@@ -1,7 +1,7 @@
-/* global polyfills not provided by core-js */
```

Comment:
Would be nice to link to https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals in the commit message since I wasn't sure what behavior this was working around and had to look it up

Reply:
Good point, done :)